cypress/support/fragments/users/userEdit.js

```diff
@@ -41,10 +41,10 @@ export default {
       cy.do(TextField('User search').fillIn(permission));
       cy.expect(TextField('User search').is({ value: permission }));
       cy.do(Button('Search').click());
-      cy.expect(MultiColumnListCell({ content: permission }).exists());
+      // wait is needed to avoid so fast robot clicks
```

Comment:
fix linter issues

Reply:
could you clarify pls what issues do you mean? I don't see it

Reply:
can't see it now